docs/dev/agent_integration.md

```diff
@@ -6,35 +6,66 @@ aliases:
   - /developers/integrations/testing/
   - /integrations/datadog_checks_dev/
   - /guides/new_integration/
-
+  - /developers/integrations/new_check_howto/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/agent_integration.md"
+further_reading:
+- link: "https://www.datadoghq.com/blog/programmatically-manage-your-datadog-integrations/"
+  tag: "Blog"
+  text: "Programmatically manage your Datadog integrations"
+- link: "/developers/integrations/python/"
+  tag: "Documentation"
+  text: "Python for Agent-based Integration Development"
+- link: "/developers/"
+  tag: "Documentation"
+  text: "Learn how to develop on the Datadog platform"
 ---
 
 ## Overview
 
-This guide provides instructions for creating a Datadog Agent integration in the `integrations-extras` repository. For more information about why you would want to create an Agent-based integration, see [Creating your own solution][1].
+This page walks Technology Partners through how to create a Datadog Agent integration, which you can list as out-of-the-box on the [Integrations page][23], or for a price on the [Marketplace page][24]. 
+
+## Agent-based integrations
+
+Agent-based integrations use the [Datadog Agent][17] to submit data through checks written by the developer. Checks can emit [metrics][23], [events][24], and [service checks][25] into a customer's Datadog account. The Agent itself can submit [logs][26] as well, but that is configured outside of the check itself. 
+
+The implementation code for these integrations is hosted by Datadog. Agent integrations are best suited for collecting data from systems or applications that live in a local area network (LAN) or virtual private cloud (VPC). Creating an Agent integration requires you to publish and deploy your solution as a Python wheel (`.whl`).
+
+You can include out-of-the-box assets such as [monitors][27], [dashboards][28], and [log pipelines][29] with your Agent-based integration. When a user clicks **Install** on your integration tile, they are prompted to follow the setup instructions, and all out-of-the-box dashboards and recommended monitors packaged with your integration will appear in the their account.
+
+## Development process
+
+The process to build an Agent-based integration looks like the following: 
+
+1. Once you've been accepted to the Datadog Partner Network, you will meet with the Datadog Technology Partner team to discuss your offering and use cases.
+2. Request a Datadog sandbox account for development.
+3. Begin development of your integration, which includes writing the integration code on your end as well as building and installing a Python wheel (`.whl`).
+4. Test your integration in your Datadog sandbox account.
+5. Once your development work is tested and complete, follow the steps in [Create a Tile][20] to display your offering on the **Integrations** or **Marketplace** page.
```

Comment:
It seems like we are still asking partners to run ddev to get the scaffolding to build an Agent integration, but then asking them on the "Create a Tile" page to rerun ddev creating another set of scaffolding. 

The scaffolding the partner creates for their Agent integration has everything they need for their tile. It'll likely be confusing to point them to documentation that asks them to create a directory run ddev again.